scrapers.js

```diff
@@ -883,39 +993,70 @@ let scrapers = [
     }
   },
   {
-    county: 'San Diego County',
-    state: 'CA',
-    country: 'USA',
-    url: 'https://www.sandiegocounty.gov/content/sdc/hhsa/programs/phs/community_epidemiology/dc/2019-nCoV/status.html',
+    county: "San Diego County",
+    state: "CA",
+    country: "USA",
+    url:
+      "https://www.sandiegocounty.gov/content/sdc/hhsa/programs/phs/community_epidemiology/dc/2019-nCoV/status.html",
     scraper: async function() {
       let $ = await fetch.page(this.url);
 
       let cases = parse.number($('td:contains("Positive (confirmed cases)")').next('td').text()) + parse.number($('td:contains("Presumptive Positive")').next('td').text());
       return {
+
         cases: cases,
         tested: parse.number($('td:contains("Total Tested")').next('td').text())
       };
     }
   },
   {
-    county: 'Ventura County',
-    state: 'CA',
-    country: 'USA',
-    url: 'https://www.ventura.org/covid19/',
+    county: "Ventura County",
+    state: "CA",
+    country: "USA",
+    url: "https://www.ventura.org/covid19/",
     // Needs JavaScript to populate counts
     scraper: async function() {
       let $ = await fetch.page(this.url);
 
       let cases = 0;
+
       cases += parse.number($('.count-subject:contains("Positive travel-related case")').closest('.hb-counter').find('.count-number').text());
       cases += parse.number($('.count-subject:contains("Presumptive Positive")').closest('.hb-counter').find('.count-number').text());
 
+
       return {
         cases: cases,
-        tested: parse.number($('.count-subject:contains("People tested")').closest('.hb-counter').find('.count-number').text()),
+        tested: parse.number(
+          $('.count-subject:contains("People tested")')
+            .closest(".hb-counter")
+            .find(".count-number")
+            .text()
+        )
       };
     }
   },
+  {
+    state: "SD",
+    country: "USA",
+    url: "https://doh.sd.gov/news/Coronavirus.aspx#SD",
+    scraper: async function() {
+      let counties = [];
+      let $ = await fetch.page(this.url);
+      let $th = $('h2:contains("South Dakota Counties with COVID-19 Cases")');
+      let $table = $th.next("table");
+      let $trs = $table.find("tbody > tr");
+
+      $trs.each((index, tr) => {
+        let $tr = $(tr);
+        counties.push({
+          county: parse.string($tr.find("> *:first-child").text()) + " County",
+          cases: parse.number($tr.find("> *:last-child").text())
+        });
+      });
+      console.log(`here's South Dakota ${counties} `);
+      return counties;
+    }
+  }
```

Comment:
Your scraper looks good. Can you fix your editor config so it doesn't make everything doublequotes and mess with long lines? Update your PR and I can merge this.

Also, please do use singlequotes to match the rest of the repo; it's less shift key pressing :)

Reply:
yup I'll kill this one so I can sync my fork up first anyway to avoid the current conflicts and add in an editor config